pkl/private/pkl_codegen_java.bzl

```diff
@@ -51,16 +53,24 @@ def _pkl_codegen_impl(ctx):
     _zipit(
         ctx = ctx,
         outfile = ctx.outputs.out,
-        files = [outdir],
+        srcs = [outdir],
     )
 
     # Return JAR
     return OutputGroupInfo(out = [outjar])
 
-_pkl_codegen = rule(
-    _pkl_codegen_impl,
+_pkl_java_jar = rule(
```

Comment:
I think I would prefer `_pkl_java_src_jar` or `_pkl_java_srcjar` to convey the fact it's made of sources. Just a suggestion.